pkg/operator/operator.go

```diff
@@ -63,9 +65,11 @@ func NewOpenShiftControllerManagerOperator(
 	namespacedKubeInformers.Core().V1().ServiceAccounts().Informer().AddEventHandler(c.eventHandler())
 	namespacedKubeInformers.Core().V1().Services().Informer().AddEventHandler(c.eventHandler())
 	namespacedKubeInformers.Apps().V1().Deployments().Informer().AddEventHandler(c.eventHandler())
+	namespacedOperatorInformers.Core().V1().ConfigMaps().Informer().AddEventHandler(c.eventHandler())
```

Comment:
@deads2k not sure if I should also be watching deployments here?

Reply:
> @deads2k not sure if I should also be watching deployments here?

I wasn't expecting a new trigger here.  This is the workload resource control loop. You aren't touching this one.